Resources/views/Resetting/email.txt.twig

```diff
@@ -0,0 +1,14 @@
+{% trans_default_domain 'FOSUserBundle' %}
+{% block subject %}
+Reset Password - CampaignChain
```

Comment:
I agree. However in a future version it would be better to make all e-mail templates configurable and translatable.